### PR TITLE
#ticket-22555555 - added information on page about us.

### PR DESCRIPTION
**Pull request by scostagit**
ticket #22555555

### CODE REVIEW

**Comment by scostagit** [comment]
llkkl

File: aboutus.html [hunk 1]
```diff
@@ -2,7 +2,11 @@
 <body>
 
 <h1>file created by contributor</h1>
-
+<div>
+    <p>We are a big company that can help you in yours Project.
+       Contact us for more information.
```
Comment (svcosta) [inline aboutus.html]: who ask you about this changes?

File: aboutus.html [hunk 2]
```diff
@@ -2,7 +2,11 @@
 <body>
 
 <h1>file created by contributor</h1>
-
+<div>
```
Comment (svcosta) [inline aboutus.html]: edgtdgdgd

**Comment by scostagit** [comment]
jljlj

**Pull request merged**